src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,147 @@
+package duke;
+
+import duke.exceptions.*;
```

Comment:
Be careful on wildcard imports. There is a settings in IntelliJ that you could set to prevent it from automatically updating the imports to wildcard that you could look out for.